internal/generate/olm-catalog/csv_updaters.go

```diff
@@ -229,20 +231,64 @@ var _ csvUpdater = crds{}
 // apply will only make a new spec.customresourcedefinitions.owned element for
 // a type if an annotation is present on that type's declaration.
 func (us crds) apply(csv *olmapiv1alpha1.ClusterServiceVersion) error {
-	ownedCRDs := []olmapiv1alpha1.CRDDescription{}
+	ownedDescs := []olmapiv1alpha1.CRDDescription{}
```

Comment:
Will revert this back to the current [`internal/scaffold/olm-catalog/csv_updaters.go`](https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/olm-catalog/csv_updaters.go) if it really is isolated from the rest of the generator changes.

Reply:
And add a TODO to move this to `internal/generate/olm-catalog/csv_updaters.go` later.

Reply:
@estroz Gave it a shot but that just causes more refactoring since if I move back csv_descriptors.go back to `internal/scaffold/olm-catalog` (and the descriptors pkg as well) then I have to refactor and expose all the structs like [`type crs [][]byte`](https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/olm-catalog/csv_updaters.go#L267) in csv_decsriptors to be public since now they're being used by `internal/generate/olm-catalog/csv.go`.

So either I move the entire pkg or not at all. So these changes will have to stay I think.